src/main/java/org/myrobotlab/programab/BotInfo.java

```diff
@@ -28,32 +29,41 @@
    * base64 png
    */
   public String img;
- 
+
   public BotInfo(ProgramAB programab, File path) {
     this.name = path.getName();
     this.path = path;
-    this.programab = programab;              
+    this.programab = programab;
     programab.info("found bot %s", name);
     properties.getProperties(FileIO.gluePaths(path.getAbsolutePath(), "config/properties.txt"));
     log.info("loaded properties");
   }
 
-  
   /**
    * task to save predicates and getting responses will eventually call getBot
    * we don't want initialization to create 2 when only one is needed
+   * 
    * @return
    */
   public synchronized Bot getBot() {
     if (bot == null) {
       // lazy loading of bot - created on the first use
-      
       if (properties.containsKey("locale")) {
         bot = new Bot(name, path.getAbsolutePath(), java.util.Locale.forLanguageTag(properties.get("locale")));
       } else {
         bot = new Bot(name, path.getAbsolutePath());
       }
 
+      // merge properties - potentially there are 2 sets
```

Comment:
Where do these other properties come from and how are they managed?  This seems unrelated to localization.

Reply:
these are not the properties your looking for ... a programab pr will be coming up .. let it go...